pkg/cmd/server/start/start_master.go

```diff
@@ -699,20 +699,66 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 
 	glog.Infof("Started Kubernetes Controllers")
 
-	// no special order
+	openshiftControllerContext := origincontrollers.ControllerContext{
+		KubeControllerContext: controllerContext,
+		ClientBuilder: origincontrollers.OpenshiftControllerClientBuilder{
+			ControllerClientBuilder: controller.SAControllerClientBuilder{
+				ClientConfig:         restclient.AnonymousClientConfig(&oc.PrivilegedLoopbackClientConfig),
+				CoreClient:           oc.PrivilegedLoopbackKubernetesClientsetExternal.Core(),
+				AuthenticationClient: oc.PrivilegedLoopbackKubernetesClientsetExternal.Authentication(),
+				Namespace:            bootstrappolicy.DefaultOpenShiftInfraNamespace,
+			},
+		},
+		DeprecatedOpenshiftInformers: oc.Informers,
+		Stop: controllerContext.Stop,
+	}
+	openshiftControllerInitializers, err := oc.NewOpenshiftControllerInitializers()
+
+	allowedOpenshiftControllers := sets.NewString(
+		"deployer",
+		"deploymentconfig",
+		"deploymenttrigger",
```

Comment:
names of the controllers... eventually we will probably create wrapper for apps.RunController (run all controllers in apps group because users don't want to disable just one deployment controller)